src/test/java/uk/gov/hmcts/reform/managecase/BaseTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.managecase;
+
+import org.junit.Before;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.context.SecurityContextImpl;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.gov.hmcts.reform.managecase.service.ras.RoleAssignmentServiceHelper;
+import uk.gov.hmcts.reform.managecase.security.SecurityUtils;
+import uk.gov.hmcts.reform.managecase.service.common.UIDService;
+
+import javax.inject.Inject;
+import java.io.IOException;
+
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+@ActiveProfiles("test")
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(locations = "classpath:test.properties")
+public abstract class BaseTest {
```

Comment:
This class and `WireMockBaseTest` are both for use in integration tests. In ACA we already have equivalents to these set up in `integrationTest` and `test` is just used for unit tests. Could we move the integration tests to the same place please?

Reply:
moved the class and deleted the classes repeated